packages/core/src/lib/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,202 @@
+// importing boilerplate stuff
+import React from 'react';
+import { composeStories } from '@storybook/testing-react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import * as stories from './Slider.stories';
+
+// // importing fire event from RTL to mock a click event
+// import { fireEvent } from '@testing-library/react';
+
+// importing color and a testing tool to convert RGB to HEX
+import {color,rgbToHex} from '@web3uikit/styles';
+
+// importing testID from button and icon
+import { buttonTestId } from '../Button/Button.test';
+
+const iconTestId = 'test-icon';
+
+// importing my stories to test
+const { Default, InitializeRed, UnderLinedText } = composeStories(stories);
+
```

Comment:
Seems like this test file is not updated 
Could you add some valid basic tests for the component...

Reply:
will do